packages/components/releaseNotes/components.md

```diff
@@ -31,6 +36,7 @@ Components, models, actions, and utility functions for LabKey applications and p
 ### version 2.182.0
 *Released*: 15 June 2022
 * Rename `isSubfolderDataEnabled` to `isProductProjectsEnabled` in all the places.
+* Issue 45710: use clone of constant instead of constant for omitted column, so we can add to it.
```

Comment:
was this actually in 2.182.0 or just a merge misplacement?

Reply:
Ah, no. It's actually in this branch. Thanks.